shell/edit/auth/ldap/config.vue

```diff
@@ -279,157 +279,125 @@ export default {
+        <h4>{{ t('authConfig.ldap.groups') }}</h4>
         <LabeledInput
-          v-model:value="model.userEnabledAttribute"
+          v-model:value="model.groupObjectClass"
           :mode="mode"
-          :label="t('authConfig.ldap.userEnabledAttribute')"
+          :label="t('authConfig.ldap.objectClass')"
         />
-      </div>
-    </div>
-    <div class="row mb-20">
-      <div class="col span-6">
         <LabeledInput
-          v-model:value="model.groupDNAttribute"
+          v-model:value="model.groupNameAttribute"
           :mode="mode"
-          :label="t('authConfig.ldap.groupDNAttribute')"
+          :label="t('authConfig.ldap.nameAttribute')"
         />
-      </div>
-      <div class="col span-6">
         <LabeledInput
-          v-model:value="model.disabledStatusBitmask"
+          v-model:value="model.groupMemberUserAttribute"
           :mode="mode"
-          :label="t('authConfig.ldap.disabledStatusBitmask')"
+          :label="t('authConfig.ldap.groupMemberUserAttribute')"
         />
-      </div>
-    </div>
-    <div class="row mb-20">
-      <div
-        v-if="!isSamlProvider"
-        class=" col span-6"
-      >
-        <RadioGroup
-          v-model:value="model.nestedGroupMembershipEnabled"
+        <LabeledInput
+          v-model:value="model.groupSearchAttribute"
           :mode="mode"
-          name="nested"
-          class="full-height"
-          :options="[true, false]"
-          :labels="[t('authConfig.ldap.nestedGroupMembership.options.nested'), t('authConfig.ldap.nestedGroupMembership.options.direct')]"
+          :label="t('authConfig.ldap.searchAttribute')"
+        />
+        <LabeledInput
+          v-model:value="model.groupSearchFilter"
+          :mode="mode"
+          :label="t('authConfig.ldap.searchFilter')"
+        />
+        <LabeledInput
+          v-model:value="model.groupMemberMappingAttribute"
+          :mode="mode"
+          :label="t('authConfig.ldap.groupMemberMappingAttribute')"
         />
+        <LabeledInput
+          v-model:value="model.groupDNAttribute"
+          :mode="mode"
+          :label="t('authConfig.ldap.groupDNAttribute')"
+        />
+        <template
+          v-if="!isSamlProvider"
+        >
+          <RadioGroup
+            v-model:value="model.nestedGroupMembershipEnabled"
+            :mode="mode"
+            name="nested"
+            class="full-height"
+            :options="[true, false]"
+            :labels="[t('authConfig.ldap.nestedGroupMembership.options.nested'), t('authConfig.ldap.nestedGroupMembership.options.direct')]"
+          />
+        </template>
       </div>
     </div>
   </div>
 </template>
+
+<style lang="scss" scoped>
+  .schema-container {
+    display: flex;
+    gap: 1.75%;
+    flex-wrap: wrap;
+  }
+
+  .schema-column {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    min-width: 16rem;
+
+    > :not(:first-child) {
+      margin-bottom: 20px;
+    }
```

Comment:
I'm using `margin-bottom` instead of gap so that I can preserve the spacing for the Users and Groups column headers. I'm open to alternative approaches.